modules/sdk-coin-ada/src/transactionBuilder.ts

```diff
@@ -0,0 +1,55 @@
+import BigNumber from 'bignumber.js';
+import { BaseCoin as CoinConfig } from 'modules/statics/src/base';
```

Comment:
```suggestion
import { BaseCoin as CoinConfig } from '@bitgo/statics';
```

Reply:
Also, curious for these imports, why does my local environment throw an error for '@bitgo/statics' packages, but not the above? Wondering if I missed a step in environment setup?

Reply:
Hmmm... have you installed the entire repo from the root with `yarn install`?

Reply:
Most definitely have done that haha, but let me see if running once more does the trick?

Reply:
Of course this time it works -__- but glad it was an easy fix